cmd/influxd/inspect/tsm_verify.go

```diff
@@ -1,81 +1,23 @@
-// Package tsm verifies integrity of TSM files.
-package tsm
+package inspect
 
 import (
-	"flag"
-	"fmt"
 	"hash/crc32"
-	"io"
 	"os"
 	"path/filepath"
-	"text/tabwriter"
 	"time"
 	"unicode/utf8"
 
 	"github.com/influxdata/influxdb/v2/tsdb/engine/tsm1"
 	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
 )
 
-// Command represents the program execution for "influx_inspect verify".
-type Command struct {
-	Stderr io.Writer
-	Stdout io.Writer
-}
-
-// NewCommand returns a new instance of Command.
-func NewCommand() *Command {
-	return &Command{
-		Stderr: os.Stderr,
-		Stdout: os.Stdout,
-	}
-}
-
-// Run executes the command.
-func (cmd *Command) Run(args ...string) error {
-	var path string
-	fs := flag.NewFlagSet("verify", flag.ExitOnError)
-	fs.StringVar(&path, "dir", os.Getenv("HOME")+"/.influxdb", "Root storage path. [$HOME/.influxdb]")
-
-	var checkUTF8 bool
-	fs.BoolVar(&checkUTF8, "check-utf8", false, "Verify series keys are valid UTF-8")
-
-	fs.SetOutput(cmd.Stdout)
-	fs.Usage = cmd.printUsage
-
-	if err := fs.Parse(args); err != nil {
-		return err
-	}
-
-	dataPath := filepath.Join(path, "data")
-	tw := tabwriter.NewWriter(cmd.Stdout, 16, 8, 0, '\t', 0)
+var checkUTF8 bool
+var dir string
+var verbose bool
```

Comment:
I would avoid using global state for these vars. The way they're used, I think you could move them into the `NewTSMVerifyCommand()` body above where `cmd` is defined